installer/release.sh

```diff
@@ -102,7 +56,7 @@ case "$version" in
 *) die "InnoSetup requires a version that begins with a digit";;
 esac
 
-render_release_notes
+../render-release-notes.sh --output installer/ --css usr/share/git/
```

Comment:
Oops. Scratch the `--copy-css` option. I realize that you no longer need to copy that.